docs/common/istio-nftables.md

```diff
@@ -0,0 +1,237 @@
+
+```sh
+NAME                                      DISPLAY         VERSION                     REPLACES                                  PHASE
+sailoperator.v1.27.0-nightly-2025-08-15   Sail Operator   1.27.0-nightly-2025-08-15   sailoperator.v1.27.0-nightly-2025-08-14   Succeeded
+```
+
+Succeeded should appear in the sailoperator CSV PHASE column.
+
+3. Create the `istio-system` and `istio-cni` namespaces.
+
+```sh
+kubectl create namespace istio-system
+kubectl create namespace istio-cni
+```
+
+4. Create the `Istio` resource with `spec.values.global.nativeNftables=true`:
+
+```sh
+apiVersion: sailoperator.io/v1
+kind: Istio
+metadata:
+  name: default
+spec:
+  version: v1.28-alpha.24646157
+  namespace: istio-system
+  updateStrategy:
+    type: InPlace
+    inactiveRevisionDeletionGracePeriodSeconds: 30
+  values:
+    global:
+      nativeNftables: true
+```
+
+5. Create the `IstioCNI` resource with `spec.values.global.nativeNftables=true`:
+
+```sh
+kind: IstioCNI
+apiVersion: sailoperator.io/v1
+metadata:
+  name: default
+spec:
+  namespace: istio-cni
+  version: v1.28-alpha.24646157
+  values:
+    global:
+      nativeNftables: true
+```
+
+#### Install in Ambient Mode
+
+### Validation
+
+When using the `nftables` backend, you can verify the traffic redirection rules using the `nft list ruleset` command in the `istio-proxy` container. When using the nftables backend, You get the table inet rules in the istio-proxy container. The following example installs a sample application `curl` in a data plane namespace `test-ns`.
+
+```sh
+kubectl create ns test-ns
+kubectl label namespace test-ns istio-injection=enabled
+kubectl apply -n test-ns -f https://raw.githubusercontent.com/istio/istio/refs/heads/master/samples/curl/curl.yaml
+```
+
+Attach a debug container and you will get the following rules in the `istio-proxy` container:
+
+```sh
+kubectl -n test-ns debug --image istio/base --profile netadmin --attach -t -i \
+  "$(kubectl -n test-ns get pod -l app=curl -o jsonpath='{.items..metadata.name}')"
+root@curl-6c88b89ddf-kbzn6:$ nft list ruleset
+
+```
+
+Verify the connectivity between two pods is working. For example, deploy a httpbin application using the following step:
+
+```sh
+kubectl apply -n test-ns -f samples/httpbin/httpbin.yaml
+kubectl exec -n test-ns "$(kubectl get pod -l app=curl -n test-ns -o jsonpath={.items..metadata.name})" -c curl -n test-ns -- curl http://httpbin.test-ns:8000/ip -s -o /dev/null -w "%{http_code}\n"
+
+200
+```
+
+More guidelines: [Debugging Guidelines](https://github.com/istio/istio/tree/master/tools/istio-nftables/pkg#debugging-guidelines)
+
+### Upgrade
+
+The migration of using the existing Istio iptables backend to nftables backend can be done by upgrading Istio. The following example installs an Istio control plane with the iptables backend and a sample application curl in a data plane namespace test-ns.
+
+```sh
+istioctl install -y
```

Comment:
As this is a Sail Operator doc, lets use the installation mechanisms supported in Sail Operator.